pgxn/neon/file_cache.c

```diff
@@ -1370,21 +1370,26 @@ neon_get_lfc_stats(PG_FUNCTION_ARGS)
 				value = lfc_ctl->size;
 			break;
 		case 5:
+			key = "file_cache_used";
```

Comment:
file_cache_used was already returned at position 2, see above

What we needed here was to actually expose this metric via sql_exporter, see files like `lfc_used.sql` and `lfc_used.libsonnet` and how they are used. We need the same for `lfc_used_pages`